### PR TITLE
[iOS] Table Horizontal Alignment Update

### PR DESCRIPTION
**Pull request by jwoo-msft**
# Description

Updated the table to use NSLayoutGuide instead of the positional anchor..

# How Verified

Verified manually by testing all table jsons.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6401)

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm [hunk 1]
```diff
@@ -133,6 +136,17 @@ - (ACRHorizontalAlignment)horizontalContentAlignment
     return ACRLeft;
 }
 
+- (ACRContainerStyle)style
```
Comment (almedina-ms) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm:139]: >style [](http://example.com/codeflow?start=21&length=5)

rename to getLastStyle or peekStyleContext or something similar

**Review by almedina-ms — APPROVED** [review]
:shipit:

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm (hunk 1 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm:139]: @almedina-ms, this is a getter for the property style. for setter of a property, set + property name.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm [hunk 2]
```diff
@@ -56,6 +58,7 @@ - (instancetype)init
         _firstRowsAsheadersContext = [[NSMutableArray alloc] init];
         _verticalAlignmentContext = [[NSMutableArray alloc] init];
         _horizontalAlignmentContext = [[NSMutableArray alloc] init];
+        _styleContext = [[NSMutableArray alloc] init];;
```
Comment (paulcam206) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACORenderContext.mm:61]: nit - extra `;`

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTableCellRenderer.mm [hunk 3]
```diff
@@ -20,4 +26,41 @@ + (ACRCardElementType)elemType
...
+               rootView:rootView
+                 inputs:inputs
+          withCardElems:cellElement->GetItems()
+          andHostConfig:acoConfig];
+
+    [cell setClipsToBounds:NO];
+
+    std::shared_ptr<BaseActionElement> selectAction = cellElement->GetSelectAction();
+    ACOBaseActionElement *acoSelectAction = [ACOBaseActionElement getACOActionElementFromAdaptiveElement:selectAction];
+    [cell configureForSelectAction:acoSelectAction rootView:rootView];
```
Comment (paulcam206) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTableCellRenderer.mm:55]: just so I'm sure: no negative consequences for a null selectaction here?

**Review by paulcam206 — APPROVED** [review]

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTableCellRenderer.mm (hunk 3 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTableCellRenderer.mm:55]: no, the method will do null check.

**Pull request merged**